cardano-node/src/Cardano/Node/Tracing/Tracers/StartLeadershipCheck.hs

```diff
@@ -52,6 +53,9 @@ data TraceStartLeadershipCheckPlus =
 forgeTracerTransform ::
   (  IsLedger (LedgerState blk)
   ,  LedgerQueries blk
+#if __GLASGOW_HASKELL__ >= 906
```

Comment:
```suggestion
#if __GLASGOW_HASKELL__ >= 960
```

Reply:
I do not think that is correct. `ghc-8.10.*` was `810`.

Reply:
I'm an idiot, thanks for enlightening me.